src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -30,6 +30,24 @@ public bool IsVerboseEnabled
             }
         }
 
+        [Event(5, Message = "Backend response {1} was not parsed. Some items may be dropped: {0}.", Level = EventLevel.Warning)]
+        public void BreezeResponseWasNotParsedWarning(string exception, string response, string appDomainName = "Incorrect")
+        {
+            this.WriteEvent(5, exception ?? string.Empty, response ?? string.Empty, this.ApplicationName);
```

Comment:
They are unique event ids. EventsSource is our unified mechanizm for tracing internal telemetry and there are certain rules how you do that. One is that you need to provide event is and you cannot use consts in the attribete so you just use same number in the attribute and inside the method.